src/quote/Quote.ts

```diff
@@ -19,6 +19,16 @@ import Identity from '../identity/Identity'
 import { IQuote, IQuoteAgreement, IQuoteAttesterSigned } from '../types/Quote'
 import { hashObjectAsStr, verify } from '../crypto/Crypto'
 
+/**
+ * Validates the quote against the meta schema and quote data against the provided schema.
+ *
+ * @param schema A [[Quote]] schema object.
+ * @param validate [[Quote]] data to be validated against the provided schema.
+ * @param messages The validating function provides an array of errors, in the case of validation failures.
+ *
+ * @returns Boolean to provide a check against the data.
```

Comment:
Suggestion:
Use the same phrasing as for boolean return values elsewhere in the SDK, e.g. "Whether the quote schema is valid".

Reply:
Agreed